main.css

```diff
@@ -78,31 +90,67 @@ p {
   margin-right: 45px;
   color: #fff;
 }
+
 /* Media Starts Below */
 
 /* Mobile Styles */
+
 @media only screen and (max-width: 400px) {
   body {
     color: black;
-    background-color: #FFCB77; /* Yellow */
+    background-color: #FFCB77;
+    /* Yellow */
     font-weight: normal;
     font-size: 14px;
   }
+  /* Logo should do a 180 spin when hovered */
+  .front {
+    transform: rotate3d(180deg);
+    transition: 2s;
+  }
+  .logo:hover {
+    transform: rotateY(180deg);
+  }
+  .logo:hover::after {
+    transform: rotateY(-180deg);
+  }
 }
 
 /* Tablet Styles */
+
 @media only screen and (min-width: 401px) and (max-width: 960px) {
   body {
-    background-color: #60B2E5; /* Blue*/
+    background-color: #60B2E5;
+    /* Blue*/
+  }
+  /* Logo should flip on the y axis */
+  .logo {
+    transform: rotateY(180deg);
+    transition-duration: 1s;
```

Comment:
🙃 This should probably match up with the duration of the other animations.

Reply:
Yep, didn't even consider that. Thx and changing that now too